<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>928</width>
    <height>517</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Cavitome Project</string>
  </property>
  <property name="windowOpacity">
   <double>18.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome to the Cavitome Project!&lt;/p&gt;&lt;p&gt;Start by selecting a local PDB file or fetch a PDB code from the RCSB PDB.&lt;/p&gt;&lt;p&gt;You may specify a chain identifier to target a specific chain (A for chain A, AB for chains A and B).&lt;/p&gt;&lt;p&gt;A custom PDB file will be generated and written in ./caviar_out/, containing the detected cavities.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="OutputTextBrowser"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_resultslog">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Results and log</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_input">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Input options</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="_3" stretch="0,1,1">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QLineEdit" name="input_filename">
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="button_browse">
              <property name="text">
               <string notr="true">Browse...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_filename">
              <property name="text">
               <string notr="true">Filename</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_PDBcode">
              <property name="text">
               <string>PDB code to fetch from RCSB PDB</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="code"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_chainID">
              <property name="text">
               <string>Chain identifier (e.g. A, AB, ACI)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="chain_id"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="exclude_missing">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Exclude cavities with missing atoms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="exclude_interchain">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Exclude interchain cavities</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_outputpymol">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <underline>true</underline>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>PyMOL and .pml file options</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pymol">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Open PyMOL when the job is over</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bypharma">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Color by pharmacophore type</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="byburi">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Color by buriedness (blue to red)</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bycav">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Color by cavity</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="run">
            <property name="text">
             <string notr="true">Run</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_close">
            <property name="text">
             <string notr="true">Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
